apps/openmw/mwrender/bulletdebugdraw.cpp

```diff
@@ -7,7 +7,8 @@
 
 #include <components/debug/debuglog.hpp>
 
-#include "vismask.hpp"
+#include <components/sceneutil/visitor.hpp>
```

Comment:
Just curious why do you add visitor.hpp here?

Reply:
An oversight, fixed.

Reply:
It should be components/sceneutil/vismask.hpp, right?

Reply:
GitHub still shows an old code to keep the context. There is an "Outdated" mark for reason.

Reply:
Oh, right, sorry. I thought it showed it changed to components/sceneutil/visitor.hpp, instead of original visitor.hpp...

The new one is indeed correct. So, merge-time?